### DISCUSSION TITLE
Can you use this to evaluate a KATEX expression and return the HTML result back?

### DISCUSSION THREAD

**Discussion by designingSparks** [r0x0r/pywebview#757] [General]
[KATEX](https://katex.org/) is a Javascript library that supports rendering to an HTML string. e.g. 

```
var html = katex.renderToString("c = \\pm\\sqrt{a^2 + b^2}", {
    throwOnError: false
});
```
Is it possible to use pywebview plus the KATEX library to get the HTML result?

**Comment by r0x0r** [comment]
I see no reason why not. use load_html or something for loading your rendered result.
